source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -224,6 +219,135 @@ def test_disruption_on_journey(self):
 
             assert any(d['uri'] == 'bob_the_disruption' for d in disruptions)
 
+@dataset([("main_routing_test", ['--BROKER.rt_topics='+chaos_rt_topic, 'spawn_maintenance_worker'])])
+class TestChaosDisruptionsBlocking(ChaosDisruptionsFixture):
+    """
+    Note: it is done as a new fixture, to spawn a new kraken, in order not the get previous disruptions
+    """
+    def test_disruption_on_journey(self):
+        """
+        We send blocking disruptions and test if the blocked object is not used
+        by the journey.
+        Then we delete it and test if use the blocked object
+        """
+        self.wait_for_rabbitmq_cnx()
+        response = self.query_region(journey_basic_query)
+
+        assert "journeys" in response
+
+        self.send_chaos_disruption_and_sleep("blocking_line_disruption", "A",
+                "line", blocking=True)
+
+        response = self.query_region(journey_basic_query+ "&disruption_active=true")
+
+        links = []
+
+        def get_line_id(k, v):
+            if k != "links" or not "type" in v or not "id" in v or v["type"] != "line":
+                return
+            links.append(v["id"])
+
+        walk_dict(response, get_line_id)
+        assert all(map(lambda id_ : id_ != "A", links))
+
+        # We send a blocking disruption on line A
```

Comment:
hum the comment is not right no ?

you need to put this one on line 238 and say there that you remove the disruption